docs/UserGuide.md

```diff
@@ -84,46 +84,65 @@ Adds a student to the contact list with the student's relevant information.
 
 Format: `add n/NAME e/EMAIL m/MODULES [t/TAG]…​`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:**<br>
+
+* A student can have any number of tags (including 0).<br>
+
+* A student's module must follow XX[X]1111[X], where X is any letter, 1 is any number and values in square brackets are optional.<br>
 
-* A student can have any number of tags (including 0).<br><br>
-* A student's module must follow XX[X]1111[X], where X is any letter,
-1 is any number and values in square brackets are optional.
 * Only student's initials and module code will be capitalised on the displayed student list.<br>
-  * e.g. `n/alex yeoh`, `n/DAVID LI` will be converted to `Alex Yeoh` and `David Li` respectively.
-  * e.g. `cs2100`, `GER1000t` will be converted to `CS2100` and `GER1000T` respectively.
+  * e.g. `n/alex yeoh`, `n/DAVID LI` will be converted to `Alex Yeoh` and `David Li` respectively.<br>
+  * e.g. `cs2100`, `GER1000t` will be converted to `CS2100` and `GER1000T` respectively.<br>
+
 </div>
 
 Examples:
 * `add n/John Doe m/CS1010 e/e1234567@u.nus.edu`
 * `add n/Betsy Crowe e/e0234567@u.nus.edu m/CS2103T`
-* `add n/alex yeoh e/1234123@u.nus.edu m/GEQ1000 t/T17`
+* `add n/alex yeoh e/e1234123@u.nus.edu m/GEQ1000 t/T17`
 
-### Listing all students : `list`
+### Listing all students: `list`
 
 Shows a list of all students in the contact list.
 
 Format: `list`
 
-### Editing a student : `edit`
+### Editing a student: `edit`
 
-Modifies any part of student's information.
+Modifies any part of the student's information.
 
-Format: `edit INDEX [m/MODULE] [e/EMAIL] [t/TAG] [n/NAME]`
+Format: `edit INDEX [m/MODULE] [e/EMAIL] [t/TAG] [n/NAME]...`
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:**<br>
 
-:bulb: **Tip:**
 * The index refers to the index number shown in the displayed student list.
-* The index **must be a positive integer within the number of students in list** 1, 2, (till the index of last student)​ 
-* You can modify several parts of student information at the same time.
-* If there are two identical prefixes in one edit command, only the last one works. 
+
+* The index **must be a positive integer within the number of students in list** 1, 2, (till the index of the last student)​
+
+* If there are two identical prefixes in one edit command, only the last one works. (except for prefix `t/`)
```

Comment:
* If you have two identical prefixes with the edit command, only the last prefix will be used for the edit. (except for prefix `t/`)